commons/src/main/java/org/acme/common/ManagedHttpClient.java

```diff
@@ -17,20 +17,25 @@
  * Copyright (c) Steve Springett. All Rights Reserved.
  */
 package org.acme.common;
-
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
```

Comment:
Since I was injecting the ManagedHttpClientFactory in the AbstractMetaAnalyzer class, the ManagedHttpClient class had to be a bean as well.. but since we have agreed to use the Arc based approach, this change would not be there anyway